editor/gui/scene_tree_editor.h

```diff
@@ -58,6 +59,27 @@ class SceneTreeEditor : public Control {
 		BUTTON_UNIQUE = 9,
 	};
 
+#define COLOR_SENTINEL Color(NAN, NAN, NAN, NAN)
+	struct CachedNode {
```

Comment:
```suggestion
#define COLOR_SENTINEL Color(NAN, NAN, NAN, NAN)

	struct CachedNode {
```
Otherwise it's a bit confusing.